inventory/hosts.example

```diff
@@ -1068,6 +1068,10 @@ debug_level=2
 #openshift_master_open_ports=[{"service":"svc1","port":"11/tcp"}]
 #openshift_node_open_ports=[{"service":"svc2","port":"12-13/tcp"},{"service":"svc3","port":"14/udp"}]
 
+# Service port node range 
+# You can change the default (20000-32767) service host port range
+#openshift_node_port_range=9000-9111
```

Comment:
Make the example range the default value, remove note about default value.

Reply:
ack